core/src/main/kotlin/org/evomaster/core/search/gene/collection/TaintedMapGene.kt

```diff
@@ -0,0 +1,144 @@
+package org.evomaster.core.search.gene.collection
+
+import org.evomaster.client.java.instrumentation.shared.TaintInputName
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.search.gene.Gene
+import org.evomaster.core.search.gene.interfaces.TaintableGene
+import org.evomaster.core.search.gene.root.CompositeGene
+import org.evomaster.core.search.gene.utils.GeneUtils
+import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.mutator.genemutation.AdditionalGeneMutationInfo
+import org.evomaster.core.search.service.mutator.genemutation.SubsetGeneMutationSelectionStrategy
+
+class TaintedMapGene (
+
+    name: String,
+
+    var taintedValue : String,
+
+    var isActive : Boolean = false,
+
+    mapGene:  MapGene<*, *>? = null
```

Comment:
if this is representing what is marshalled from JSON, then keys are always strings